rskj-core/src/main/java/co/rsk/peg/BridgeEvents.java

```diff
@@ -60,6 +60,12 @@ public enum BridgeEvents {
                     new CallTransaction.Param(false, "activationHeight", SolidityType.getType(SolidityType.INT256))
             }
     ),
+    COMMIT_FEDERATION_FAILED("commit_federation_failed",
+        new CallTransaction.Param[]{
+            new CallTransaction.Param(false, "proposedFederationRedeemScriptSerialized", SolidityType.getType(SolidityType.BYTES)),
```

Comment:
Nit, but I would probably remove the `serialized` part of the name. That is pretty much implied

Reply:
updated